solidity/slither.config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "exclude_dependencies": true,
```

Comment:
Do we want to exclude dependencies? I would almost rather ignore dependency issues so that if a dependency introduces something, we can look at it and make sure we don't have to worry about it.